core/platform/auth/firebase_auth_services_test.py

```diff
@@ -496,68 +630,69 @@ def swap_delete_user_to_always_fail(self):
         with self.swap_initialize_sdk_to_always_fail():
-            self.wipeout()
+            self.delete_auth_associations()
 
         self.assert_firebase_account_is_not_deleted()
-        self.assert_wipeout_is_not_verified()
+        self.assert_auth_associations_are_not_deleted()
 
-        self.wipeout()
+        self.delete_auth_associations()
 
         self.assert_firebase_account_is_deleted()
-        self.assert_wipeout_is_verified()
+        self.assert_auth_associations_are_deleted()
 
-    def test_wipeout_retry_after_unverified_successful_attempt(self):
-        self.wipeout()
+    def test_delete_auth_associations_retry_after_unverified_successful_attempt(
+            self):
+        self.delete_auth_associations()
 
         self.assert_firebase_account_is_deleted()
 
         with self.swap_initialize_sdk_to_always_fail():
-            self.assert_wipeout_is_not_verified()
+            self.assert_auth_associations_are_not_deleted()
 
-        self.wipeout()
+        self.delete_auth_associations()
 
-        self.assert_wipeout_is_verified()
+        self.assert_auth_associations_are_deleted()
 
-    def test_wipeout_when_delete_user_fails(self):
+    def test_delete_auth_associations_when_delete_user_fails(self):
         with self.swap_delete_user_to_always_fail():
-            self.wipeout()
+            self.delete_auth_associations()
 
         self.assert_firebase_account_is_not_deleted()
 
-        self.assert_wipeout_is_not_verified()
+        self.assert_auth_associations_are_not_deleted()
 
-    def test_wipeout_when_get_user_fails(self):
-        self.wipeout()
+    def test_delete_auth_associations_when_get_user_fails(self):
+        self.delete_auth_associations()
 
         self.assert_firebase_account_is_deleted()
 
         with self.swap_get_user_to_always_fail():
-            self.assert_wipeout_is_not_verified()
+            self.assert_auth_associations_are_not_deleted()
 
-        self.assert_wipeout_is_verified()
+        self.assert_auth_associations_are_deleted()
 
-    def test_wipeout_when_init_fails_during_delete(self):
+    def test_delete_auth_associations_when_init_fails_during_delete(self):
         with self.swap_initialize_sdk_to_always_fail():
-            self.wipeout()
+            self.delete_auth_associations()
 
         self.assert_firebase_account_is_not_deleted()
 
-        self.assert_wipeout_is_not_verified()
+        self.assert_auth_associations_are_not_deleted()
 
-    def test_wipeout_when_init_fails_during_verify(self):
-        self.wipeout()
+    def test_delete_auth_associations_when_init_fails_during_verify(self):
+        self.delete_auth_associations()
 
         self.assert_firebase_account_is_deleted()
 
         with self.swap_initialize_sdk_to_always_fail():
-            self.assert_wipeout_is_not_verified()
+            self.assert_auth_associations_are_not_deleted()
 
-        self.assert_wipeout_is_verified()
+        self.assert_auth_associations_are_deleted()
```

Comment:
I'm a bit confused by the way this and some of the tests above are written because nothing happens between line 696 and 698. What is the action that actually causes the assertion to change? The calls here seem like GET-type calls that should not modify state.

Reply:
Resolved offline - the assertions have been inlined so these method names are obsolete.